doc/api_assets/style.css

```diff
@@ -27,21 +27,29 @@ h4, h4 code { font-size: 1.5rem; }
 h5, h5 code { font-size: 1.25rem; }
 h6, h6 code { font-size: 1rem; }
 
-h1, h1 code,
-h2, h2 code,
-h3, h3 code,
-h4, h4 code,
-h5, h5 code,
-h6, h6 code {
-  color: inherit;
-  font-family: inherit;
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
   font-weight: 700;
   line-height: inherit;
   position: relative;
   margin: 1.5rem 0 1rem;
   text-rendering: optimizeLegibility;
 }
 
+h1 code,
+h2 code,
+h3 code,
+h4 code,
+h5 code,
+h6 code {
+  color: inherit;
+  font-family: inherit;
```

Comment:
`font-family` can be removed later too @Trott so that the code is using the same font as everywhere else. AFAICT only the color is needed

Reply:
> `font-family` can be removed later too @Trott so that the code is using the same font as everywhere else. AFAICT only the color is needed

I thought about doing that in a second commit here, but I want to make sure there aren't any weird kerning issues or anything when we do that. So I was going to save that for another PR. I'm really good at rationalizing dividing CSS changes across multiple PRs apparently. But if you're more confident it won't be a problem, feel free to add it to this one.

Reply:
I actually meant to make it in a separate PR later. Now that the selectors are split, it should be easier to maintain.